blockchain/beacon/proto/common.proto

```diff
@@ -0,0 +1,35 @@
+/*
+ * Protobuf file containing empty message definition
+ */
+syntax = "proto3";
+
+package drand;
+
+/*option go_package = "github.com/drand/drand/protobuf/drand";*/
+option go_package = "drand";
+
+message Empty {
+
+}
```

Comment:
I believe so. I only copied over `common.proto` and `api.proto` over here from Drand. It might be used else where. I basically copied them over 1 to 1 and don't want to change them, since it'd be easier to diff with what Drand has if anything changes. I am told that they will eventually break off their proto files into another repo so we can submodule it.